cmd/cluster-agent/main.go

```diff
@@ -15,9 +15,10 @@ import (
 	"net/http"
 	"os"
 
-	_ "expvar"         // Blank import used because this isn't directly used in this file
```

Comment:
Let's keep that import to keep the collector/forwarder/AD expvars exposed on the 5000 port